app/assets/javascripts/controllers/fonticon_picker_controller.js

```diff
@@ -0,0 +1,18 @@
+/* global add_flash */
+(function() {
+  var COTNROLLER_NAME = 'fonticonPickerController';
+
+  FonticonPickerController = function($element) {
+    var vm = this;
+    // This is an ugly hack to be able to use this component in a non-angular context with miq-observe
+    // FIXME: Remove this when the form is converted to angular
+    var hidden = $($element[0]).find('input[type="hidden"]');
```

Comment:
I made this code the ugliest as possible on purpose so no one will forget to remove it when converting the form to Angular.